src/utils/todo/task-filing/content-loader.ts

```diff
@@ -183,8 +183,7 @@ export const addExpansionContent = (todoItem: TodoItemI): void => {
     // if it is a draft correction filing
     todoItem.expansionContent = TodoExpansionContentE.DRAFT_CORRECTION
   } else if (todoItem.name === FilingTypes.CORRECTION) {
-    // if it is a correction filing (non-draft)
-    todoItem.expansionContent = TodoExpansionContentE.CORRECTION
+    // if it is a non-draft correction filing (pending filing), no expansion panel in this case
   } else if (
```

Comment:
this `addExpansionContent` function determines what expansion panel to be rendered for the todo item. It covers the logic in https://github.com/bcgov/business-filings-ui/blob/f9747e4b20d50da231dcd8a45d1feb45ffa5a498/src/components/Dashboard/TodoList.vue#L589-L691   In TodoList.vue L623-L634, there is a template for a correction todo item that is not draft. However, this expansion panel is never rendered in the Filing UI. In TodoList.vue of Filing UI, the 'View Detail' button is controlled by `showDetailsBtnRed()` (L929) and `showDetailsBtnBlue()` (L900).